### PR TITLE
return the appliance address if db is None

### PR DESCRIPTION
**Pull request by amavinag**
* this will avoid to NoneType Error Exception.

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 85699199a9be8e8dcff96e42f87d311287d9b87c:

Everything seems all right :smile: .
*CFME QE Bot*

File: utils/appliance.py [hunk 1]
```diff
@@ -1359,16 +1359,39 @@ def wait_for_ssh(self, timeout=600):
...
         # pulls the db address from the appliance by default, falling back to the appliance
         # ip address (and issuing a warning) if that fails. methods that set up the internal
         # db should set db_address to something else when they do that
         try:
             if self.version >= '5.6':
-                db = self.get_yaml_config("vmdb")['server']['host']
+                db = self.wait_for_host_address('vmdb')
             else:
-                db = self.get_yaml_file('/var/www/miq/vmdb/config/vmdb.yml.db')['server']['host']
+                db = self.wait_for_host_address('/var/www/miq/vmdb/config/vmdb.yml.db')
```
Comment (mfalesni) [inline utils/appliance.py]: One last question, the `wait_for_host_address` once takes 'vmdb' and another time it takes '/var/www/...' ... is that all right?

Comment (amavinag) [inline utils/appliance.py]: @mfalesni I think that's fine as config_name is the parameter for wait_for_host_address. Let me know if it is going to affect in some other way.

Comment (psav) [inline utils/appliance.py]: mfalesni is right, this is going to break, one used ```get_yaml_file```, the other used ```get_yaml_config```, these details have been lost.

Comment (amavinag) [inline utils/appliance.py]: that's right, I completely missed the get_yaml_file(). I think we can add wait based on version and just add db=get_host_address(config_name) in line#1389

and in def get_for_host_address():
                if version >= '5.6':
                      get_yaml_config()
               else: 
                     get_yaml_file()

**Comment by dajoRH** [comment]
Lint report for commit b41d4c465f9e000419c935fd740ea9adf1394a26:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit dd0146b6adec6d9132e4b99f74960e9a77ecc00d:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by psav** [comment]
:cake::punch::cookie:

**Pull request merged**